### PR TITLE
Change servers button

### PR DESCRIPTION
**Pull request by NotFenixio**

### CODE REVIEW

File: src/components/ChangeServers.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            label="Uploads"
+            defaultValue={uploads}
+            required
+            onInput={(e) => setUploadsValue(e.currentTarget.value)}
+            value={uploadsValue}
+          />
+        </div>
+        <Button
+          onClick={() =>
+            location.replace(location.origin + location.pathname + `?api=${apiValue}&cl=${clValue}&uploads=${uploadsValue}`)
```
Comment (mybearworld) [inline src/components/ChangeServers.tsx:57]: Could these be `encodeURIComponent`ed?

**Review by mybearworld — CHANGES_REQUESTED** [review]
One small thing, looks good otherwise, thank you!

**Review by Supernoodles99 — APPROVED** [review]
hamburger

**Review by mybearworld — APPROVED** [review]
Thanks!

**Pull request merged**